static/html/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nailivic</title>
+    <script src="https://unpkg.com/htmx.org@2.0.2"></script>
+    <!-- <link rel="icon" href="/static/img/cat-space-duotone.svg"> -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Big+Shoulders+Inline+Display:wght@100..900&display=swap|family=Big+Shoulders+Text:wght@100..900&display=swap|family=Lora:ital,wght@0,400;0,700;1,400;1,700&display=swap"
```

Comment:
I know the plan is to self host the svg images. Fonts are expected to remain an external dependency (having the client fetch from google rather than our server)?

Reply:
Yes, fonts are expected to remain external. Do you have reservations about that?

Reply:
I don't think so, beyond a general "host all the things" desire for consistency.

But this is structured in a way that you can just make a one-line change to point at a self hosted backend if we build that.